scripts/bfb-install

```diff
@@ -392,11 +392,16 @@ check_nic_mode()
   # Get PCIE BDF
   str=`cat ${rshim_node}/misc | grep DEV_NAME | awk '{print $2}'`
   str=${str:5}
-  str=${str/.*/.0}
-  pcie_bdf="$str"
+  str=${str/.*/}
+  pcie_bd="$str"
+  num_ports=$(lspci -nn -s $pcie_bd | grep -v '\[0801\]' | wc -l || echo 0)
+  if [ "$num_ports" -eq 2 ]; then
+      dual_pcie_port=1
+      [ $verbose -eq 1 ] && echo "Detected dual-port device: ${pcie_bd}.0 and ${pcie_bd}.1"
+  fi
```

Comment:
Should we add  'else' to set dual_pcie_port=0? Or else line 549 below might have issue.